### DISCUSSION TITLE
Welcome to dwains-lovelace-dashboard Discussions!

### DISCUSSION THREAD

**Discussion by dwainscheeren** [dwainscheeren/dwains-lovelace-dashboard#265] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Giuseppe-P** [comment]
I have tried several times but it doesn't work. In your guide you say you have the configs folder in dwains-dashboard / configs / while the configs is located in dwains-dashboard / installation / configs /

I can not make it

**Reply by dwainscheeren** [reply]
The config is in your root home assistant folder not in custom components

**Reply by Giuseppe-P** [reply]
Thanks I didn't notice.

**Reply by Giuseppe-P** [reply]
why does it start in blue mode and i can't see the icons?  I have to close and reopen the application to see it in the correct mode

**Reply by Giuseppe-P** [reply]
how can i use personalized cards?  https://dwainscheeren.github.io/dwains-lovelace-dashboard/how-tos/custom-cards.html

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
